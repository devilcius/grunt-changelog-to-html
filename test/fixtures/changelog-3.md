## test link

## [0.1.0](https://github.com/devilcius/grunt-changelog-to-html/tree/v0.1.0) (2022-12-15)