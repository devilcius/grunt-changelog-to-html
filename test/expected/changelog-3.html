<h2 id="test-link">test link</h2>
<h2 id="010-2022-12-15"><a href="https://github.com/devilcius/grunt-changelog-to-html/tree/v0.1.0">0.1.0</a> (2022-12-15)</h2>
